src/app/home/footer/footer.component.html

```diff
@@ -0,0 +1,18 @@
+
+    <footer class="footer font-family-semibold"  >
+      <div class="d-flex" style="padding-top: 4%; padding-right:12%; padding-left:12%">
+        <div class="p-2 opacity" >Privacy Policy</div>
+        <div class="p-2 opacity">Terms of use</div>
+        <div class="p-2 opacity">Site map</div>
+        <div class="p-2 opacity">Mobile site</div>
+        <div class="ml-auto p-2">About us</div>
+        
+        <div class="p-2">Blog</div>
+        <div class="p-2">Carrers</div>
+        <div class="p-2">Press</div>
+        <div class="p-2">Advertise</div>
+      </div>
+      <div class="d-flex" style="padding-right:12%; padding-left:12%; padding-bottom: 3%">
```

Comment:
Same comment for the inline styles. Please remove inline styes and use classes instead.